api/views.py

```diff
@@ -0,0 +1,27 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import ensure_csrf_cookie
+from rest_framework import generics
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.permissions import IsAuthenticated
+
+from osis_async.api.serializers import AsyncTaskSerializer
+from osis_async.models import AsyncTask
+
+
+class AsyncTaskSetPagination(LimitOffsetPagination):
+    default_limit = 15
+
+
+@method_decorator(ensure_csrf_cookie, name='dispatch')
+class AsyncTaskListView(generics.ListAPIView):
+    """Return all the asynchronous tasks associated with a specific user."""
+
+    queryset = AsyncTask.objects.all()
+    serializer_class = AsyncTaskSerializer
+    permission_classes = (IsAuthenticated,)
+    pagination_class = AsyncTaskSetPagination
+    authentication_classes = (SessionAuthentication,)
+
+    def get_queryset(self):
+        return super().get_queryset().filter(person__uuid=self.request.user.person.uuid)
```

Comment:
Ici tu peux retirer une jointure en faisant 
```suggestion
        return super().get_queryset().filter(person_id=self.request.user.person_id)
```

Reply:
Ah en fait ça ne fonctionne pas ça :/

Reply:
'User' object has no attribute 'person_id'

Reply:
`super().get_queryset().filter(person_id=self.request.user.person.id)`
j'aurais pas du cliquer sur 'commit' sans tester \o/